doc/releases/release-notes-3.6.rst

```diff
@@ -130,6 +132,8 @@ Boards & SoC Support
 
 * Made these changes for Xtensa boards:
 
+  * Added ESP32S3 based M5Stack AtomS3 board: ``m5stack_atoms3``
```

Comment:
can you please check, this doesn't sound right at all

```
git diff --name-only --diff-filter=A v3.5.0..HEAD -- boards/ | grep \.rst -i

...
boards/xtensa/esp32s2_lolin_mini/doc/index.rst
boards/xtensa/heltec_wireless_stick_lite_v3/doc/index.rst
boards/xtensa/kincony_kc868_a32/doc/index.rst
boards/xtensa/m5stack_atoms3/doc/index.rst
boards/xtensa/m5stack_atoms3_lite/doc/index.rst
boards/xtensa/m5stack_stamps3/doc/index.rst
```

Reply:
Aha, sorry, thanks!

Reply:
wait, there's now ... zero board in your PR? :)